media/js/firefox/hello/start.js

```diff
@@ -25,9 +25,17 @@
 
     // GA event for successful Hello conversations
     function trackGAConversationConnect() {
+        // get source from localStorage
+        var tourSource = localStorage.getItem('hello_ftu_tour_source');
```

Comment:
We should wrap all uses of `localStorage` in a `try/catch`, to avoid an error should the user have cookies disabled for example.